core/templates/dev/head/domain/question/QuestionUpdateService.js

```diff
@@ -0,0 +1,95 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+* @fileoverview Service to handle the updating of a question.
+*/
+
+oppia.constant('QUESTION_PROPERTY_LANGUAGE_CODE', 'language_code');
+oppia.constant('QUESTION_PROPERTY_QUESTION_STATE_DATA', 'question_state_data');
+
+oppia.constant('CMD_UPDATE_QUESTION_PROPERTY', 'update_question_property');
+
+oppia.factory('QuestionUpdateService', [
+  'QuestionObjectFactory',
```

Comment:
I think 2 or 3 dependencies can be declared in a line?

Reply:
done